sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/SqlIoTest.java

```diff
@@ -540,6 +540,43 @@ public void compare(String query, String expected, boolean optimize) {
         this.addRustTestCase(query, compiler, circuit, streams);
     }
 
+    // Unsure what to do with `data`
```

Comment:
if the argument is unused it shouldn't exist. The only exception is if you implement an interface which requires the argument.